### PR TITLE
[13.0] [MIG] web_editor_background_color: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT25341

Here you have an example how it works

Please @Yajo @Tardo can you review this

### CODE REVIEW

File: web_editor_background_color/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (victoralmau) [inline web_editor_background_color/__manifest__.py]:
```suggestion
# Copyright 2017 Tecnativa -  Jairo Llopis
```

File: web_editor_background_color/i18n/ca.po [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+"Last-Translator: Marc Tormo i Bochaca <mtbochaca@gmail.com>, 2017\n"
+"Language-Team: Catalan (https://www.transifex.com/oca/teams/23907/ca/)\n"
+"Language: ca\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#~ msgid "Color"
+#~ msgstr "Color"
```
Comment (victoralmau) [inline web_editor_background_color/i18n/ca.po]:
```suggestion
```

File: web_editor_background_color/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+"Language-Team: \n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.8.6\n"
+
+#~ msgid "Pick Background Color"
+#~ msgstr "Escoger el color de fondo"
```
Comment (victoralmau) [inline web_editor_background_color/i18n/es.po]:
```suggestion
```

File: web_editor_background_color/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+"Language-Team: French (http://www.transifex.com/oca/OCA-social-8-0/language/"
+"fr/)\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#~ msgid "Pick Background Color"
+#~ msgstr "Mettre la couleur de fond"
```
Comment (victoralmau) [inline web_editor_background_color/i18n/fr.po]:
```suggestion
```

File: web_editor_background_color/i18n/sl.po [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+"language/sl/)\n"
+"Language: sl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=4; plural=(n%100==1 ? 0 : n%100==2 ? 1 : n%100==3 || n"
+"%100==4 ? 2 : 3);\n"
+
+#~ msgid "Pick Background Color"
+#~ msgstr "Izbira barve ozadja"
```
Comment (victoralmau) [inline web_editor_background_color/i18n/sl.po]:
```suggestion
```

File: web_editor_background_color/static/src/css/background_color.scss [hunk 6]
```diff
@@ -0,0 +1,14 @@
+/* Copyright 2017-2019 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (victoralmau) [inline web_editor_background_color/static/src/css/background_color.scss]:
```suggestion
/* Copyright 2017-2019 Tecnativa - Jairo Llopis
```

File: web_editor_background_color/static/src/js/background_color.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
+/* Copyright 2016-2019 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (victoralmau) [inline web_editor_background_color/static/src/js/background_color.js]:
```suggestion
/* Copyright 2016-2019 Tecnativa - Jairo Llopis
```

File: web_editor_background_color/templates/assets.xml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017-2019 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (victoralmau) [inline web_editor_background_color/templates/assets.xml]:
```suggestion
<!-- Copyright 2017-2019 Tecnativa - Jairo Llopis
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good job!! Some little changes.

**Comment by CarlosRoca13** [comment]
@victoralmau changes done!

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
It doesn't work for mass mailings:

Peek 18-11-2020 10-29

**Comment by CarlosRoca13** [comment]
@pedrobaeza Problem solved!

**Comment by pedrobaeza** [comment]
It's still not working correctly. It doesn't work until you select first time another color, and then the custom color is applied, but if you try to set the custom color directly, it's not set.

**Comment by CarlosRoca13** [comment]
I don't see what you say... It could be that you haven't changed the opacity? The example that I paste is from runbot

**Comment by pedrobaeza** [comment]
Yeah, that can definitively be, but then the next question is: why it appears with opacity = 0 by default? Is it the same in previous versions?

**Comment by CarlosRoca13** [comment]
Yes, in v12 works like here

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**